Docs/sphinx_doc/VerticalSCoord.rst

```diff
@@ -0,0 +1,93 @@
+
+ .. role:: cpp(code)
+    :language: c++
+
+Vertical S-coordinate
+=====================
+
+.. _'ROMS documentation': https://www.myroms.org/wiki/Vertical_S-coordinate
+
+These equations are as given in the 'ROMS documentation'_
+
+ROMSX has a generalized vertical, terrain-following, coordinate system.  Currently, the vertical transformation equation :math:'z=z\left( x,y,\sigma ,t\right) ', is available which can support vertical stretching 1D-functions when several constraints are satisfied.
+
+Transformation Equation
+-----------------------
+
+.. math::
+   z\left( x,y,\sigma ,t\right) &=\zeta \left( x,y,t\right) +\left[ \zeta \left( x,y,t\right) +h\left( x,y\right) \right] S\left( x,y,\sigma \right) ,\\
+   S\left( x,y,\sigma \right) &=\frac{h_c \sigma +h \left( x,y\right) C\left( \sigma \right) }{h_c +h\left( x,y\right)}
+
+where :math:'S\left( x,y,\sigma \right)' is a nonlinear vertical transformation functional, :math:'\zeta \left( x,y,t\right)' is the time-varying free-surface, :math:'h\left( x,y\right) ' is the unperturbed water column thickness and :math:'z=-h\left( x,y\right) ' corresponds to the ocean bottom, :math:'\sigma ' is a fractional vertical stretching coordinate ranging from :math:'-1\leq \sigma \leq 0,C\left( \sigma \right) ' is a nondimensional, monotonic, vertical stretching function ranging from :math:'-1\leq C\left( \sigma \right) \leq 0', and :math:'h_c' is a positive thickness controlling the stretching.  In sediment applications, :math:'h=h\left( x,y,t\right) ' is changed at every time-step since it is affected by erosion and deposition processes.
```

Comment:
Good to have the full text, but maybe comment out the sediment part as that's not currently implemented in ROMSX

Reply:
I wondered.

Reply:
done